app-template/template-setup.sh

```diff
@@ -1,10 +1,5 @@
 #!/usr/bin/env bash
 
-# Add the CI to this repo
-git remote add ci https://github.com/AbstractSDK/rust-ci
-git fetch ci
-git merge ci/main --allow-unrelated-histories
-
```

Comment:
It gives an error on your machine? What if we update `rust-ci` repo, just merge it everywhere manually?

Reply:
We don't really need this because the CI is already included in the template dir. We use the ci repo for ease of pushing out changes to the different rust repos that need CI.